src/Directory.Build.props

```diff
@@ -1,9 +1,10 @@
 <Project>
   <PropertyGroup>
     <Copyright>Copyright © Groupe GSoft inc. $([System.DateTime]::UtcNow.ToString(yyyy))</Copyright>
```

Comment:
`GSoft Group Inc.` at lines 3, 4, and 5

Reply:
Ohh sneaky one, you're right hadn't noticed that